Program.cs

```diff
@@ -92,15 +93,19 @@ public static void Main(string[] args)
 
             string version = $"{verBase}{increment}";
 
+            var targetPlatform = RuntimeInfo.OS;
+
             if (args.Length > 1 && !string.IsNullOrEmpty(args[1]))
                 version = args[1];
+            if (args.Length > 2 && !string.IsNullOrEmpty(args[2]))
+                Enum.TryParse(args[2], true, out targetPlatform);
 
             Console.ResetColor();
             Console.WriteLine($"Increment Version:     {IncrementVersion}");
             Console.WriteLine($"Signing Certificate:   {CodeSigningCertificate}");
             Console.WriteLine($"Upload to GitHub:      {GitHubUpload}");
             Console.WriteLine();
-            Console.Write($"Ready to deploy {version}!");
+            Console.Write($"Ready to deploy version {version} on platform {targetPlatform}!");

```

Comment:
I am not a native English speaker but is "deploy on platform" correct? Shouldn't it be "deploy for platform"? According the the PR description, you can deploy on a different platforms than you are building for.

Reply:
It's fine really.